Assets/Scripts/Data/Pickup Pup/GiftDatabase.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*
+ * Authors: Grace Barrett-Snyder, James Hostetler
+ * Description: Stores data about the gifts in the game 
+ */
+
+using UnityEngine;
+using System.IO;
+
+
```

Comment:
I believe we're only using one newline between `using` statements and classname (though it's not explicitly documented in the TDD)